purchase_ext_sst/views/purchase_order_views.xml

```diff
@@ -1,12 +1,33 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
+    <record id="purchase_order_user_purchase_order_form_inherit"
+            model="ir.ui.view">
+        <field name="name">purchase.order.form.inherit</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id"
+               ref="purchase_order_user.purchase_order_form_inherit"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='user_id']" position="replace"/>
+        </field>
+    </record>
+
+    <record id="po_so_field_purchase_order_form_inherit" model="ir.ui.view">
+        <field name="name">purchase.order.form.inherit</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="po_so_field.purchase_order_form_inherit"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='sale_order_id']" position="replace"/>
```

Comment:
This made my dev env threw server error since a view in `po_convert_so` module references this field.  We should make it invisible instead of replacing it.